docs/user/docs/settings/tools/capella/t4c-git-compare.md

```diff
@@ -0,0 +1,77 @@
+<!--
+ ~ SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+ ~ SPDX-License-Identifier: Apache-2.0
+ -->
+
+# Ways of co-working on a Capella project
+
+## Quick comparision
+
+<table markdown="1">
+  <tr>
+    <th></th>
+    <th>Git</th>
+    <th>TeamForCapella</th>
+  </tr>
+  <tr>
+    <td><b>Multiple workstreams</b></td>
+    <td>Different workstreams available via branches</td>
+    <td>Not recommended, only possible via different repositories</td>
+  </tr>
+  <tr>
+    <td><b>Concurrent working</b></td>
+    <td>Very challenging because of merge conflict potential.</td>
+    <td>Is possible. Diagrams are locked, no risk of merge conflicts.</td>
+  </tr>
+  <tr>
+    <td><b>Merge conflicts</b></td>
+    <td>Resolution of merge conflicts can be challenging.</td>
+    <td>No merge conflicts when one workstream is used. For multiple workstreams, the resolution of merge conflicts can be challenging.</td>
+  </tr>
+  <tr markdown="1">
+  <!-- This formatting is required, otherwise mkdocs breaks -->
+  <td><b>License</b></td>
+  <td markdown="1">No license needed[^1]</td>
+  <td markdown="1">TeamForCapella license required[^2]</td>
+  </tr>
+  <tr>
+    <td><b>Complexity</b></td>
+    <td>High complexity, Git knowledge is required.</td>
```

Comment:
The complexity is not high, as long as you can stay with commit/push/pull only.

Reply:
> as long as you can stay with commit/push/pull only

Our projects have shown that this is more of a wishful thinking. Sooner or later at least a merge will be necessary.
I tried to clarify it a bit more - "High complexity" was removed, it just says "Git knowledge is required". A footnote has been added with the following content:

```
For the basic workflow, a simple Git knowledge that includes commit, pull
    and push is sufficient. However, it is necessary to have at least one Git
    expert in the project to deal with potential merge conflicts and unexpected
    situations.
```